src/cluster/slot_migrate.h

```diff
@@ -133,52 +123,51 @@ class SlotMigrate : public Redis::Database {
   Status MigrateStream(const rocksdb::Slice &key, const StreamMetadata &metadata, std::string *restore_cmds);
   Status MigrateBitmapKey(const InternalKey &inkey, std::unique_ptr<rocksdb::Iterator> *iter,
                           std::vector<std::string> *user_cmd, std::string *restore_cmds);
+
   Status SendCmdsPipelineIfNeed(std::string *commands, bool need);
   void ApplyMigrationSpeedLimit() const;
   Status GenerateCmdsFromBatch(rocksdb::BatchResult *batch, std::string *commands);
   Status MigrateIncrementData(std::unique_ptr<rocksdb::TransactionLogIterator> *iter, uint64_t end_seq);
-  Status SyncWalBeforeForbidSlot();
-  Status SyncWalAfterForbidSlot();
+  Status SyncWalBeforeForbiddingSlot();
```

Comment:
Should we also change `Wal` to `WAL` for keeping consistent with previous function `SyncWAL`

Reply:
Done